app/src/main/java/org/jellyfin/androidtv/ui/playback/VideoManager.java

```diff
@@ -98,14 +98,16 @@ public VideoManager(PlaybackOverlayActivity activity, View view) {
         mSubtitlesSurface.setZOrderMediaOverlay(true);
         mSubtitlesSurface.getHolder().setFormat(PixelFormat.TRANSLUCENT);
 
-        mExoPlayer = new ExoPlayer.Builder(TvApp.getApplication(), new DefaultRenderersFactory(TvApp.getApplication()) {
-            @Override
-            protected void buildTextRenderers(Context context, TextOutput output, Looper outputLooper, int extensionRendererMode, ArrayList<Renderer> out) {
-                // Do not add text renderers since we handle subtitles
-            }
-        }).build();
-
-
+        DefaultRenderersFactory defaultRendererFactory =
+                new DefaultRenderersFactory(activity) {
+                    @Override
+                    protected void buildTextRenderers(Context context, TextOutput output,
+                                                      Looper outputLooper, int extensionRendererMode,
+                                                      ArrayList<Renderer> out) {
+                        // Do not add text renderers since we handle subtitles
+                    }
+                }.setExtensionRendererMode(DefaultRenderersFactory.EXTENSION_RENDERER_MODE_PREFER);
```

Comment:
Why `EXTENSION_RENDERER_MODE_PREFER` instead of `EXTENSION_RENDERER_MODE_ON`? (just wondering)

Reply:
It was to match the factory in the [Android repository](https://github.com/jellyfin/jellyfin-android/blob/1fe181baa069aaa1e440403b26aa90bf492cc51b/app/src/main/java/org/jellyfin/mobile/player/PlayerViewModel.kt#L165)

But I've only partially transplanted it, good spot I'll take this block as-is from the other repo. 
That way we can rely having the same FFMPEG extensions and configuration on mobile and TV would help us share work between the two long-term too

Reply:
Since we can't access the media queue, I've opted to let Exoplayer try to handle this internally with `setEnableDecoderFallback(true)`

I'd like to gradually bring them into sync, but since we don't have a `MediaQueue`, and this still works for non-native formats I'll leave that as a future exercise